client/src/components/Upload/helpers.js

```diff
@@ -1,6 +1,15 @@
 export const defaultNewFileName = "New File";
 
-const URI_PREFIXES = ["http", "https", "ftp", "file", "gxfiles", "gximport", "gxuserimport", "gxftp"];
+const URI_PREFIXES = [
+    "http://",
+    "https://",
+    "ftp://",
+    "file://",
```

Comment:
we should definitely not allow file:// as a protocol, right? especially not for user facing anything (or is that needed for 'import from path')

Reply:
That I don't know, but the authoritative check is done on the backend, this just determines what looks like a URL vs what should be assumed is pasted content.

Reply:
Ah, fine then, no worries. (I'm pretty sure I checked that when I did the local address filtering, like 7 years ago, so, we're fine)